setup.cfg

```diff
@@ -18,6 +18,8 @@ zip_safe = false
 packages = find_namespace:
 python_requires = >=3.8
 tests_require = pytest
+setup_requires = 
```

Comment:
I don't think this is needed here, packaging is used on githubactions only to patch version numbers before creates wheels/sources not when using those to install the package.

Reply:
Yeah I wasn't sure if that was still needed.  If you think it isn't I'll remove it again.